tiledb/sm/misc/types.h

```diff
@@ -138,8 +138,11 @@ class Range {
 
   /** Copies 'start' into this range's start bytes for fixed-size ranges. */
   void set_start(const void* const start) {
-    if (var_size_)
-      LOG_FATAL("Unexpected var-sized range; cannot set end range.");
+    if (var_size_) {
+      LOG_ERROR("Unexpected var-sized range; cannot set start range.");
+      return;
```

Comment:
I can't see any side effects produced in case for instance `set_end` fails in `add_or_coalesce_range`. But this doesn't mean it won't produce a silent bug in the future if `add_or_coalesce_range` code changes, so I agree with you. Should I push a followup fix for this @eric-hughes-tiledb? Thanks for the review

Reply:
I think it would be a good idea to have a two-line PR to throw here. It's very likely to be forgotten otherwise.